apps/projects/app/components/App/App.js

```diff
@@ -372,15 +374,17 @@ class App extends React.PureComponent {
       'onReviewWork',
       web3.toHex(issue.repoId),
       issue.number,
-      issue.assignee,
-      state.accepted
+      issue.workSubmissions[issue.workSubmissions.length - 1],
+      state.accepted,
+      issue.workSubmissions[issue.workSubmissions.length - 1].submissionIPFSHash
     )
     this.closePanel()
     this.props.app.reviewSubmission(
       web3.toHex(issue.repoId),
       issue.number,
-      issue.assignee,
-      state.accepted
+      issue.workSubmissions.length - 1,
+      state.accepted,
```

Comment:
Not needed for this branch but need a denied flow.

Reply:
rejecting a submission updates the submission's onchain state to "rejected" and that updated state is now pulled into the frontend. just needs to be displayed now.

Reply:
`state.accepted` is a boolean that is set depending on whether **Accept** or **Reject** is clicked